.travis.yml

```diff
@@ -40,11 +40,8 @@ install:
 - export PATH=$HOME/.local/bin:$PATH;
 
 # install openmdao and sphinx-travis
-- git clone https://github.com/openmdao/openmdao.git;
-- cd openmdao;
+- pip install `openmdao==3.1.0`
```

Comment:
Do we need to specify 'openmdao==3.1.0' or can we instead leave it without a specific version so that when openmdao is updated in the future we do not need to change our yml file to have travis catch issues?

Reply:
I'm fine with not pinning the version and just relying on pip. That way, if anything breaks we'll know, and it's also consistent with stating that we support v3.1.0+.

Reply:
Request to change:
`pip install 'openmdao==3.1.0'`
To:
`pip install 'openmdao'`